mesonbuild/backend/ninjabackend.py

```diff
@@ -455,7 +455,7 @@ def generate_target(self, target, outfile):
         # This will be set as dependencies of all the target's sources. At the
         # same time, also deal with generated sources that need to be compiled.
         generated_source_files = []
-        for rel_src, gensrc in generated_sources.items():
+        for rel_src, _ in generated_sources.items():
```

Comment:
actually, better than this would be:
```python
for rel_src in generated_sources.keys():
```

Reply:
Yeah, I also should have thought about that. Thanks.

Reply:
`for rel_src in generated_sources:` works as well.

Reply:
Sure but using `.keys()` makes it obvious that we are iterating over a dict and not a list.